### ISSUE TITLE
Why Succes?

### ISSUE THREAD

**Issue by valbers** [fjoppe/Legivel#19]
What's wrong with "Success"?

**Comment by fjoppe** [comment]
There is nothing wrong with Success - they say success is a choice ;)

I suppose you've found a typo? "Succes" is the dutch spelling, my native language.

**Comment by FoggyFinder** [comment]
```fs
type DeserializeResult<'tp> =
    |   Succes of SuccessInfo<'tp>
    |   Error of ErrorInfo
```

https://github.com/fjoppe/Legivel/blob/3f49f146f6d2cad8eca8944d624ca62631287ac4/src/Legivel.Mapper/YamlMapper.fs#L56

**Comment by valbers** [comment]
Oh, I didn't know it is Dutch. It's a shame because it looks like a misspelling of the English word. Whatever, it must be one of those fun facts of the software about things which were introduced once upon a time and nobody cares if it stays this way.

**Comment by valbers** [comment]
I think it'd be cool to have a side note about it somewhere in the "Getting started" documentation. This "typo" right away feels like a clumsy project.

**Comment by fjoppe** [comment]
I'm sorry, I did not want to sound refusive. This typo is an issue. Because the current spelling is in my native language, I've been blind to it. But it is _supposed_ to be English. So it is an issue and I'll fix it. Will  be my first breaking change!

**Comment by fjoppe** [comment]
Fixed in v0.4.0 - rename causes breaking change.

https://www.nuget.org/packages/Legivel/0.4.0
